src/Eppie.CLI/Eppie.CLI/Resources/Program.resx

```diff
@@ -230,6 +238,14 @@
     <value>Enter your new vault password: </value>
     <comment>This message prompts the user to enter a new vault password.</comment>
   </data>
+  <data name="Message.AskRestorePath" xml:space="preserve">
+    <value>Enter the restore URI: </value>
+    <comment>This message prompts the user to enter the path to the backup.</comment>
+  </data>
+  <data name="Menu.RestoreDescription" xml:space="preserve">
+    <value>Restore your master account</value>
```

Comment:
Could you please change to Eppie account?

Reply:
done at [commit](https://github.com/Eppie-io/Eppie-CLI/pull/82/commits/392e5e3a11abf4947162a6a833ac17e0319efa6a)